core/domain/user_services.py

```diff
@@ -551,3 +556,16 @@ def get_user_impact_score(user_id):
         return model.impact_score
     else:
         return 0
+
+def fetch_gravatar(email):
+    """Returns the gravatar corresponding to given email"""
```

Comment:
(The above is a suggested docstring. I suggest also changing the body of the function to reflect this docstring, and adding tests for this function.)